extensions/oidc/runtime/src/main/java/io/quarkus/oidc/SecurityEvent.java

```diff
@@ -11,8 +11,17 @@
  */
 public class SecurityEvent extends AbstractSecurityEvent {
     public static final String SESSION_TOKENS_PROPERTY = "session-tokens";
+    public static final String AUTH_SERVER_URL = "auth-server-url";
 
     public enum Type {
+        /**
+         * OIDC endpoint access event which is reported if an attempt to connect to the OIDC server failed.
+         */
+        OIDC_SERVER_NOT_AVAILABLE,
+        /**
+         * OIDC endpoint access event which is reported if connection with the OIDC server has been recovered.
```

Comment:
```suggestion
         * OIDC connection event which is reported when a connection with the OIDC server has been established.
```

Reply:
I used `OIDC endpoint access` because that was name of the exception thrown by OIDC, personally couldn't care less, will use your version.